tests/leapYear.test.ts

```diff
@@ -0,0 +1,7 @@
+import { leapYear } from "../src/leapYear";
+
+describe("When I check if 1996 is a leap year", () => {
+  it("should return 'true'", () => {
+    expect(leapYear()).toBe(true);
```

Comment:
I would never have wrote the test like this
I know theres the idea that you should write the minimum code to make the test pass but I think you should have passed the year into the function immediately